tools/ds-identify

```diff
@@ -179,10 +179,30 @@ debug() {
     echo "$@" 1>&3
 }
 
+map_sysfs_to_dmi() {
+    case "$@" in
+        sys_vendor) _RET="system-manufacturer";;
+        product_name) _RET="system-product-name";;
+        product_uuid) _RET="system-uuid";;
+        product_serial) _RET="system-serial-number";;
+        chassis-asset-tag) _RET="chassis_asset_tag";;
```

Comment:
this seems wrong.
or at least its inconsistent.  Pretty sure current code calls 'get_dmi_field chassis_asset_tag'.

Note the underscores and dashes.

Reply:
it looks like the two sides are swapped  🤨